modules/distr-tracing-tempo-config-spanmetrics.adoc

```diff
@@ -0,0 +1,91 @@
+////
+This module included in the following assemblies:
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+////
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-spanmetrics_{context}"]
+= Configure monitor tab in Jaeger UI
+
+Trace data contains rich information and most importantly the data is normalized across instrumented languages and frameworks.
+Therefore, additional metrics can be extracted from traces. These metrics are request count, duration and error count (RED).
+The metrics can be visualized in Jaeger console in the Monitor tab.
```

Comment:
As a reader, I'm assuming that there is a tab titled "Monitor". Let me know if this isn't correct.
```suggestion
The metrics can be visualized in Jaeger console in the *Monitor* tab.
```